e2e/runner/sample-apps-shared/helpers/start-containers.ts

```diff
@@ -41,11 +41,13 @@ export async function startContainers({
     shell(dockerDownCommand, { cwd, env });
   }
 
-  ["exit", "SIGINT", "SIGTERM", "uncaughtException"].forEach(signal => {
-    process.on(signal, () => {
-      console.log(`Parent received ${signal}, killing stopping app...`);
+  if (!process.env.CI) {
```

Comment:
why do we need to exclude CI? Does our CI send any of these signals and kill the app unintentionally or something?